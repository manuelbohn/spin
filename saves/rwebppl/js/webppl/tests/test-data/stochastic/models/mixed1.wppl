var model = function() {
  var x = beta(1, 2);
  var y = flip(x);
  return y;
};
