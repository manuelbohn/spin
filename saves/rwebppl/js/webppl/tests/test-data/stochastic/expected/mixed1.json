{
  "hist": {
    "true": 0.333,
    "false": 0.667
  }
}
